tensorflow/python/keras/callbacks.py

```diff
@@ -1102,11 +1102,13 @@ class ModelCheckpoint(Callback):
         in the filename.
       monitor: quantity to monitor.
       verbose: verbosity mode, 0 or 1.
-      save_best_only: if `save_best_only=True`, the latest best model according
-        to the quantity monitored will not be overwritten.
-        If `filepath` doesn't contain formatting options like `{epoch}` then
-        `filepath` will be overwritten by each new better model.
-      mode: one of {auto, min, max}. If `save_best_only=True`, the decision to
+      save_best_only: if `save_best_only=True`, not only the "latest best model" 
+        according to the quantity monitored will not be overwritten, but also 
```

Comment:
Thanks for the PR! I feel the updated description becomes a bit confusing. Can we say in the first sentence of `save_best_only` arg doc that it only saves when the model is considered the "best", and keep the original doc?

Reply:
Sure! Thank you for reviewing! How does this sound to you? @rchao 
  ```
save_best_only: if `save_best_only=True`, it only saves 
when the model is considered the "best" and the latest 
best model according to the quantity monitored will not 
be overwritten. If `filepath` doesn't contain formatting 
options like `{epoch}` then `filepath` will be overwritten 
by each new better model.
```

Reply:
fixed!